### DISCUSSION TITLE
Letsencrypt renewal

### DISCUSSION THREAD

**Discussion by snarum** [requarks/wiki#4654] [General]
Hi

Could you change the days before letsencrypt certificate is renewed? It looks to me that the certificate is automatically renewed 5 days before expiry. Letsencrypt recommends 1/3 of certificate length i.e. 30 days. 10 days before expiration Letsencrypt sends out warning email, so this should definately be more than 10 days.

Thanks for a great piece of software!

**Comment by semik** [comment]
Certificate on mine wiki just expired, looking at it's code I've found: https://github.com/requarks/wiki/blob/9dddef3a3fceac5134a261ee8a0f937568585cf6/server/core/letsencrypt.js#L21 but I do not think this is being called periodically to make certificate renewal. To me it looks this is called only during wiki start. 

See https://github.com/requarks/wiki/issues/2945#issuecomment-972551253 there @NGPixel states that renewal isn't working. :(
